iot/manager/manager_test.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2018 Google LLC. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+	"testing"
+
+	"cloud.google.com/go/pubsub"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+	"github.com/google/uuid"
+)
+
+var topicID string
+var projectID string
+
+var client *pubsub.Client
+
+func TestMain(m *testing.M) {
+	setup()
+	log.SetOutput(ioutil.Discard)
+	s := m.Run()
+	log.SetOutput(os.Stderr)
+	shutdown()
+	os.Exit(s)
+}
+
+func setup() {
+	ctx := context.Background()
+	// Retrieve project ID from console
+	projectID = os.Getenv("GOLANG_SAMPLES_PROJECT_ID")
```

Comment:
Please use `testutil.SystemTest` to get the projectID.

Example: https://github.com/GoogleCloudPlatform/golang-samples/blob/master/kms/asymmetric/samples_test.go#L101